test/unit/processing/payment/get-settlement-value.test.js

```diff
@@ -0,0 +1,27 @@
+const getSettledValue = require('../../../../app/processing/payment/calculate-scheduled-payments/get-settlement-value')
+const previousSettlements = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-settlements')))
+describe('get settled value', () => {
+  beforeEach(() => {
+
+  })
+
+  test('returns maximum value', () => {
```

Comment:
Add test  for empty array

Reply:
Good spot for the empty array test.

The best test in this scenario is the behaviour of the final output (schedule object) from combination of various messages from different subscriptions. The test frame-work that the test-team is working on would help with behavioural test, and would also, enable component integration and message injection test, that would help to identify behaviour of  schedule generated.

Reply:
Can we not do something in our integration tests as they are now?

We should have (probs out of scope for this) an integration test from `app/processing/process-statements.js` to test how a statement is generated and sim. to schedule

Reply:
Created task for comprehensive integration test for `get-payment-schedule.js `

https://dev.azure.com/defragovuk/DEFRA-FFC/_backlogs/backlog/DEFRA-FFC%20Documents%20Team/Stories/?workitem=182645